test/test_models.py

```diff
@@ -94,6 +94,7 @@ def test_save_to_model():
 
 
 def test_unique():
+    install_labels(User)
     User(email='jim1@test.com', age=3).save()
     try:
         User(email='jim1@test.com', age=3).save()
```

Comment:
This is a round-about way of checking that an exception was raised. Pytest has a more readable way of doing this:

```
with pytest.raises(neomodel.exceptions.ConstraintValidationFailed) as e:   
    User(email='jim1@test.cim', age=3).save()
```

**EDIT:** Maybe the exception is `UniqueProperty` here but you get the idea.

Reply:
Updated